src/test/java/rx/internal/operators/OperatorMergeMaxConcurrentTest.java

```diff
@@ -218,7 +219,11 @@ public void testSimpleAsync() {
     }
     @Test(timeout = 10000)
     public void testSimpleOneLessAsyncLoop() {
-        for (int i = 0; i < 200; i++) {
+        int max = 200;
+        if (PlatformDependent.isAndroid()) {
+            max = 50;
```

Comment:
Hah :) Though, I guess it'll fail on Travis CI because emulator runs really slow on it.